oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyState.java

```diff
@@ -19,24 +19,16 @@
 import static java.util.Collections.emptyList;
 import static org.apache.jackrabbit.oak.plugins.memory.PropertyStates.createProperty;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
 import java.util.List;
 
 import javax.jcr.PropertyType;
 
 import org.apache.jackrabbit.guava.common.collect.Lists;
```

Comment:
we might want to remove this guava usage as well, while we're effectively changing this class entirely.

Reply:
Done.